### ISSUE TITLE
Error with new tone feature when doing a research

### ISSUE THREAD

**Issue by r7l** [assafelovic/gpt-researcher#657]
Using the most recent commit it causes the generated report output to fail with this error:

```
  File "/projects/researcher/backend/server.py", line 68, in websocket_endpoint
    report = await manager.start_streaming(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/researcher/backend/websocket_manager.py", line 56, in start_streaming
    report = await run_agent(task, report_type, report_source, tone, websocket)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/researcher/backend/websocket_manager.py", line 88, in run_agent
    report = await researcher.run()
             ^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/researcher/backend/report_type/basic_report/basic_report.py", line 41, in run
    report = await researcher.write_report()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/researcher/gpt_researcher/master/agent.py", line 179, in write_report
    report = await generate_report(
             ^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/researcher/gpt_researcher/master/actions.py", line 345, in generate_report
    content = f"{generate_prompt(query, context, report_source, report_format=cfg.report_format, tone=tone, total_words=cfg.total_words)}"
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: generate_report_prompt() got an unexpected keyword argument 'tone'
```

The attribute seems to be missing in the function.

**Comment by fbruzzaniti** [comment]
Same

**Comment by bruno-uy** [comment]
@0x11c11e you removed the `tone` parameter from `generate_report_prompt` in the PR #654, but the commit aa6718ff94214aaa0231634f3c8b0ce04b482312 has that. I'm not sure what happened, but should be good if you can restore the changes from the commit I mentioned.

**Comment by 0x11c11e** [comment]
Thank you for the report. The function needs to include Tone. I will create a branch now and address this issue.

**Comment by assafelovic** [comment]
Thanks @0x11c11e merged fix

**Comment by bruno-uy** [comment]
Solved with #659. @r7l can you close the issue?

**Comment by r7l** [comment]
Yes, tests and it works now. Thanks allot!
